resources/resincli.robot

```diff
@@ -100,6 +102,40 @@ Remove ENV variable ${variable_name} from application ${application_name}
     ${result} =  Run Process    resin env rm ${result_id.stdout} --yes     shell=yes
     Process ${result}
 
+Set up loop device
```

Comment:
I am wondering if it makes sense to move all of these to a resinos.robot resource file ? Because these dont belong in the resincli.robot as they dont use the cli at all.